wallet-new/src/Cardano/Wallet/Kernel/Restore.hs

```diff
@@ -71,7 +71,7 @@ import           Pos.Util.Trace (Severity (Error))
 --
 -- Wallet initialization parameters match those of 'createWalletHdRnd'
 restoreWallet :: Kernel.PassiveWallet
-              -> Bool -- ^ Spending password
+              -> PassPhrase
```

Comment:
Let's do this: let's keep the `Bool` here, but also add an extra argument to pass an `Address` which can be used to initialise the `defaultHdAddress`.